versions.props

```diff
@@ -1,5 +1,5 @@
 org.slf4j:* = 1.7.25
-org.apache.avro:avro = 1.9.2
+org.apache.avro:avro = 1.10.0
```

Comment:
Can we bump this to Avro 1.10.1? I was specifically interested in https://issues.apache.org/jira/browse/AVRO-2817. I can also create a followup PR if you prefer and leave it at 1.10.0 here.

Reply:
I've bumped it to 1.10.1